src/orb-tools/orb.yml

```diff
@@ -1,9 +1,149 @@
 version: 2.1
+
 orbs:
   cli: circleci/circleci-cli@volatile
+
+commands:
+  validate:
+    description: >
+      Users the CLI to validate a given orb yml
+    parameters:
+      orb-path:
+        description: "Path to the orb to be validated."
+        type: string
+    steps:
+      - run:
+          name: "Validate whether this is a well-formed orb."
+          command: circleci orb validate << parameters.orb-path >>
+
+  pack:
+    description: >
+      *Experimental*
+      Uses the CLI to pack an orb file structure
+      into a single orb yml.
+    parameters:
+      source:
+        description: "Path to the root folder of an unpacked orb."
+        type: string
+      destination:
+        description: "Path including the filename to output the packed orb."
+        type: string
+    steps:
+      - run:
+          name: "Pack << parameters.source>>"
+          command: |
+            circleci config pack << parameters.source >> > << parameters.destination >>
+
+  persist-orb-to-workspace:
```

Comment:
This feels like a lot of noise with little value -- it's many lines of code to do a single line of code. We should talk about orb design and think about whether exposing commands that are purely a pass-through to a single "command" is worth it.

Reply:
I think it's on the edge of DRY. I was on the fence about pulling this out, but given it's not purely a pass-though, as it does ensure the workspace dir exists and copies the orb over, I decided to pull it out. We also currently use it in 2 jobs and I could see it possibly being used for more.

In contrast `attach_workspace` was not pulled out into a command because of exactly what you described.